src/controllers/tasks/people.with.significant.control.controller.ts

```diff
@@ -148,6 +153,33 @@ const formatPSCForDisplay = (psc: PersonOfSignificantControl): PersonOfSignifica
 
   clonedPsc.serviceAddressLine1 = formatTitleCase(psc.serviceAddressLine1);
   clonedPsc.serviceAddressPostTown = formatTitleCase(psc.serviceAddressPostTown);
+  clonedPsc.serviceAddressCareOf = formatTitleCase(psc.serviceAddressCareOf);
+  clonedPsc.serviceAddressCountryName = formatTitleCase(psc.serviceAddressCountryName);
+  clonedPsc.serviceAddressPoBox = formatTitleCase(psc.serviceAddressPoBox);
+  clonedPsc.serviceAddressArea = formatTitleCase(psc.serviceAddressArea);
+  clonedPsc.serviceAddressRegion = formatTitleCase(psc.serviceAddressRegion);
 
   return clonedPsc;
 };
+
+const formatServiceAddress = (formattedPsc: PersonOfSignificantControl): string => {
```

Comment:
can you move the format functions into utils/format.ts? should clean up the controller a bit then